plugins/ltac/g_rewrite.mlg

```diff
@@ -207,27 +207,6 @@ VERNAC COMMAND EXTEND AddRelation3 CLASSIFIED AS SIDEFF
       { declare_relation atts a aeq n None None (Some lemma3) }
 END
 
-{
-
-type binders_argtype = local_binder_expr list
-
-let wit_binders =
- (Genarg.create_arg "binders" : binders_argtype Genarg.uniform_genarg_type)
-
-let binders = Pcoq.create_generic_entry2 "binders" (Genarg.rawwit wit_binders)
-
-let () =
-  let raw_printer env sigma _ _ _ l = Pp.pr_non_empty_arg (Ppconstr.pr_binders env sigma) l in
-  Pptactic.declare_extra_vernac_genarg_pprule wit_binders raw_printer
```

Comment:
You should still declare the printer.

Reply:
Done.

I added also the missing vernac printer for `red_expr` (or do you prefer a different PR?).

Reply:
No it's fine, it can be considered part of the same PR.